### DISCUSSION TITLE
PickList that locks source list and enable target repeating elements (taken from source list)

### DISCUSSION THREAD

**Discussion by danielemaddaluno** [primefaces/community#1106] [PrimeFaces] [answered]
Is there a way to obtaining this behaviour with the `p:pickList`?
Looking at this pickList demo page: [Primefaces Picklist](http://www.primefaces.org:8080/showcase/ui/data/pickList.xhtml) I need a way to lock source list and enable target repeating elements (taken from source list).
Is there a simple way to obtain this behaviour?
If the answer is positive, could you please make a code example?

**Comment by melloware** [comment, marked as answer]
@danielemaddaluno see here: https://github.com/primefaces/primefaces/issues/11249 its been on the Picklist wishlist for years and will probably have to be financially sponsored.

Original issue: https://github.com/primefaces/primefaces/issues/5309
